pixsfm/base/src/interpolation.h

```diff
@@ -139,6 +133,17 @@ class Interpolator {
     }
   }
 
+
+    // The following two Evaluate overloads are needed for interfacing
+    // with automatic differentiation. The first is for when a scalar
+    // evaluation is done, and the second one is for when Jets are used.
+    template <>
```

Comment:
Explicit specialization in class scope is accepted by clang but not by GCC:
> error: explicit specialization in non-namespace scope ‘class pixsfm::Interpolator<Grid>’

This is part of the standard only since C++17 and still a bug in GCC: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=85282 Will look into this when I find the time.